lms/templates/courseware/course_about.html

```diff
@@ -319,6 +319,14 @@
             </li>
           % endif
 
+          % if pre_requisite_course:
+          <li class="prerequisite-course important-dates-item">
+            <i class="icon icon-list-ul"></i>
+            <p class="important-dates-item-title">${_("Requirements")}</p>
```

Comment:
@srpearce "Prerequisites" area in the body of the page is coming from "Overview" section of "Sechdule & Details" page on studio. so I can not change that text However, I can change "Requirements" text on the sidebar to "Prerequisites".